crates/redstone/src/crypto/mod.rs

```diff
@@ -1,7 +1,12 @@
+use crate::{Bytes, SignerAddress};
 use alloc::vec::Vec;
 use core::fmt::Debug;
+use primitive_types::U256;
 
-use crate::{Bytes, SignerAddress};
+const ECDSA_N_DIV_2: [u8; 32] = [
```

Comment:
```suggestion
const ECDSA_N_DIV_2: U256 = U256([16134479119472337056, 6725966010171805725, 18446744073709551615, 9223372036854775807]);
```

We could also add a test that its the same as U256::from(hex_repr); just to be sure